src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -142,24 +113,22 @@ public void apply() throws ConfigurationException {
 
   @Override
   public void reset() {
-    isModified = false;
-
     final FlutterSdk sdk = FlutterSdk.getGlobalFlutterSdk();
-    final String path = sdk != null ? FileUtilRt.toSystemDependentName(sdk.getHomePath()) : "";
+    final String path = sdk != null ? sdk.getHomePath() : "";
     sdkCombo.getComboBox().getEditor().setItem(path);
 
     updateVersionText();
     updateErrorLabel();
   }
 
   private void updateVersionText() {
-
     final FlutterSdk sdk = FlutterSdk.forPath(getSdkPathText());
     if (sdk == null) {
       versionDetails.setVisible(false);
     }
     else {
       try {
+        final ModalityState modalityState = ModalityState.current();
```

Comment:
Got it. You probably use IntelliJ IDEA 2016.2 and assertion was added later, see [this commit](https://github.com/JetBrains/intellij-community/commit/abbffc933ff652660146eeb4731818eeeb340ddb).
`idea.log` tab is probably new in 2016.3 as well (but in 2016.2 exceptions appear in Console tab). So you do not see exception, but it doesn't mean that you use ModalityState correctly :).